app/models/pafs_core/state.rb

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 module PafsCore
   class State < ActiveRecord::Base
+    VALID_STATES = %w[draft completed submitted updatable updated archived finished].freeze
+
     belongs_to :project, inverse_of: :state
-    validates :state, inclusion: { in: %w[draft completed submitted updatable updated archived finished] }
+    validates :state, inclusion: { in: VALID_STATES }
```

Comment:
Could we move this in to a separate PR :)

Reply:
Yeah - this ended up here as I found that the endpoint wasn't being tested. I'll move it to a separate PR. Do I need a separate JIRA ticket too @baphled?

Reply:
Nah, diff PR should be cool :)